### PR TITLE
[671] Improve support of root Namespace

### PR DESCRIPTION
**Pull request by gdaniel**
Bug: https://github.com/eclipse-syson/syson/issues/671

### CODE REVIEW

**Review by AxelRICHARD — COMMENTED** [review]
Could you add some cypress tests for the new filter and the "new object" menu please ?

File: backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/configuration/SysONTreeFilterProvider.java [hunk 1]
```diff
@@ -35,12 +35,15 @@ public class SysONTreeFilterProvider implements ITreeFilterProvider {
...
 
+    public static final String HIDE_ROOT_NAMESPACE_ID = UUID.nameUUIDFromBytes("SysONTreeRootNamespaceFilter".getBytes()).toString();
+
     @Override
     public List<TreeFilter> get(String editingContextId, TreeDescription treeDescription, String representationId) {
         List<TreeFilter> filters = new ArrayList<>();
         filters.add(new TreeFilter(HIDE_MEMBERSHIPS_TREE_ITEM_FILTER_ID, "Hide Memberships", true));
         filters.add(new TreeFilter(HIDE_KERML_STANDARD_LIBRARIES_TREE_FILTER_ID, "Hide KerML Standard Libraries", true));
         filters.add(new TreeFilter(HIDE_SYSML_STANDARD_LIBRARIES_TREE_FILTER_ID, "Hide SysML Standard Libraries", true));
+        filters.add(new TreeFilter(HIDE_ROOT_NAMESPACE_ID, "Hide Root Namespace", true));
```
Comment (AxelRICHARD) [inline backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/configuration/SysONTreeFilterProvider.java:46]: Hide Root Namespace => Hide Root Namespaces

**Review by AxelRICHARD — COMMENTED** [review]
The creation of objects from documents seems to not work anymore.
Could you please check again and add a cypress test about that?

File: backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/configuration/SysONTreeFilterProvider.java (hunk 1 above)
Comment (gdaniel) [reply backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/configuration/SysONTreeFilterProvider.java]: Fixed

File: integration-tests/cypress/workbench/Explorer.ts [hunk 2]
```diff
@@ -47,19 +47,27 @@ export class Explorer {
     cy.getByTestId('insert-textual-sysmlv2-submit').click();
   }
 
-  public createRootObject(documentTreeItemLabel: string, domain: string, entity: string) {
+  public getFilter(filterLabel: string): Cypress.Chainable<JQuery<HTMLElement>> {
+    cy.getByTestId('tree-filter-menu-icon').click();
+    cy.getByTestId('tree-filter-menu').should('be.visible');
+    // tree-filter-menu-checkbox-Hide Root Namespaces
```
Comment (AxelRICHARD) [inline integration-tests/cypress/workbench/Explorer.ts:53]: please remove these comments

Comment (gdaniel) [reply integration-tests/cypress/workbench/Explorer.ts]: done

**Pull request merged**